src/Umbraco.Tests.UnitTests/Umbraco.Core/Models/DataTypeTests.cs

```diff
@@ -4,7 +4,7 @@
 using Umbraco.Tests.Common.Builders;
 using Umbraco.Tests.Common.Builders.Extensions;
 
-namespace Umbraco.Tests.UnitTests.Umbraco.Infrastructure.Models
+namespace Umbraco.Tests.UnitTests.Umbraco.Core.Models
```

Comment:
One would think that due to the similarities with the other Test classes that you successfully moved from <code>Umbraco.Tests.UnitTests.Umbraco.<b>Infrastructure</b>.Models</code> to <code>Umbraco.Tests.UnitTests.Umbraco.<b>Core</b>.Models</code> this file can also be migrated in a similar manner.

However, this is not the case here 🙂 

- Even though `DataType` class has a `Umbraco.Core` namespace, it is located in _Umbraco.Infrastructure_ folder
  - It also has a _Newtonsoft.Json_ using, meaning that we are going to keep it in this location.

At this point, we don't have all the correct namespaces but we should trust the files location

Reply:
Good spot - have moved to the right place now.